sdk/src/wallet/account/operations/retry.rs

```diff
@@ -119,7 +119,17 @@ where
                 // After we checked all our reattached blocks, check if the transaction got reattached in another block
                 // and confirmed
                 if conflicting {
-                    let included_block = self.client().get_included_block(transaction_id).await?;
+                    let included_block = self.client().get_included_block(transaction_id).await.map_err(|e| {
+                        if matches!(
+                            e,
+                            crate::client::Error::Node(crate::client::node_api::error::Error::NotFound(_))
```

Comment:
You could import at least `crate::client::Error` to make this shorter

Reply:
Happy with the change?

Reply:
Sort of. Still feels like it could be cleaner somehow, but it's fine 🤷‍♀️